manifests/server/params.pp

```diff
@@ -6,6 +6,7 @@
   $broker_id                           = false
   $log_dirs                            = ['/var/spool/kafka']
   $zookeeper_connect                   = false
+  $broker_rack			       = false
```

Comment:
@ecanzonieri I am not sure why this is showing in github diff as misaligned. On my dev box it is not showing , I suspect it might be because of the '+'?

`class kafka::server(

  $enable                              = $kafka::server::params::enable,

  # Essential Kafka configuration
  $broker_id                           = $kafka::server::params::broker_id,
  $log_dirs                            = $kafka::server::params::log_dirs,
  $zookeeper_connect                   = $kafka::server::params::zookeeper_connect,
  $broker_rack                         = $kafka::server::params::broker_rack,
`